src/NuGet.Core/NuGet.Protocol.Core.Types/ProtocolConstants.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace NuGet.Protocol
+{
+    public static class ProtocolConstants
+    {
+
+        public static readonly string ApiKeyHeader = "X-NuGet-ApiKey";
+        public static readonly string ServerWarningHeader = "NuGet-Warning";
```

Comment:
as much as I hate `X-` headers, lets keep it consistent with `X-NuGet-ApiKey`. Thoughts?